src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -232,7 +232,7 @@ def forward(
             value = torch.cat((past_value, value), dim=-2)
 
         if use_cache is True:
-            present = torch.stack((key.transpose(-2, -1), value))  # transpose to have same shapes for stacking
+            present = (key.transpose(-2, -1), value)  # transpose to have same shapes
```

Comment:
This is the reason for the recent failure of the slow test:

`RUN_SLOW=1 pytest tests/test_onnx.py::OnnxExportTestCase::test_export_pytorch` 

Can you fix the onnx part easily? @mfuntowicz @Narsil